### ISSUE TITLE
Unable to create rules for Microsoft Teams

### ISSUE THREAD

**Issue by mrjohnjones** [henrypp/simplewall#1271]
The only way for me to successfully run the desktop app is to disable simplewall or just allow everything out defeating its purpose. Does anyone have any suggestions?

I'd just simple disable when I need Teams, but with WFH becoming more prevalent, this is becoming an issue and I'm not sure of a good solution.

**Comment by Altefier** [comment]
Many Microsoft apps rely on a lot of other apps and services to have an internet connection or else they refuse to work. I don't know what can be done to figure out what these programs want, unfortunately

**Comment by cvionnet** [comment]
Just for information, it seems that I have the same issue as @mrjohnjones since 2 weeks.
Teams is launching, but only display some parts of the UI and it ends with a message displaying that the internet connection is offline. Maybe a bug in Teams, it would not be the first.

Same solution : disable filters in Simplewall > launch Teams > wait 10 sec that it connects wherever it needs > enable filters in Simplewall

(Windows 11 & Teams desktop are up-to-date)

**Comment by neilgrz** [comment]
I attempted to try process of elimination by having a rule for the Teams.exe (all protocols, ports, etc. allowed) and temporarily allowing everything in Services, UWP apps, and disabled all Blocklists. However, even after all that, Teams still did not authenticate. That surprised me because I did expect it to work with at least several of those allowed, but with essentially everything allowed, it still failed.
